libs/ast-guard/docs/STDLIB-SECURITY.md

```
@@ -0,0 +1,762 @@
+
+**3. Object.defineProperty()**
+
+```javascript
+Object.defineProperty(Object.prototype, 'polluted', {
+  value: true,
+});
+```
+
+**Status:** ❌ Blocked by not whitelisting defineProperty
+
+**4. JSON.parse() with **proto\*\*\*\*
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)